yo/blockchain_follower.py

```diff
@@ -1,82 +1,170 @@
-from .base_service import YoBaseService 
-from .db import notifications_table,PRIORITY_LEVELS
+from .base_service import YoBaseService
+from .db import notifications_table, PRIORITY_LEVELS
 import asyncio
 import steem
 from steem.blockchain import Blockchain
 import json
 
 import logging
+
 logger = logging.getLogger(__name__)
 
 # TODO - use reliable stream when merged into steem-python
 
 # Basically this service just follows the blockchain and inserts into the DB then triggers the notification sender to send the actual notification
 
+# NOTIFICATION TYPES
+ACCOUNT_UPDATE = 'account_update'
+# Not a blockchain event ANNOUNCEMENT_IMPORTANT = 'announcement_important'
```

Comment:
This could be a blockchain event if we listen for a `custom_json` broadcasted by e.g. `@steemit`

Reply:
I think that particular notification type is included for future uses only - idea being we can inject marketing/announcement notifications via the API and not just from the blockchain